inc/fleet.class.php

```diff
@@ -533,7 +534,11 @@ public function refreshPersistedNotifications() {
       ];
       foreach ($DB->request($request) as $row) {
          $task->getFromDB($row['id']);
-         $task->publishPolicy($this);
+         try {
+            $task->publishPolicy($this);
+         } catch (TaskPublishPolicyPolicyNotFoundException $exception) {
+            // we shall not send technical information to the error message.
+         }
```

Comment:
PHP CS will complain. If you want to send a message to the end-user, use a user friendly message. If you want to keep a log, feel free to use Toolbox::logInFile()